tap_klaviyo/utils.py

```diff
@@ -106,14 +106,15 @@ def get_starting_point(stream, state, start_date):
 def get_latest_event_time(events):
     return ts_to_dt(int(events[-1]['timestamp'])) if len(events) else None
 
-@backoff.on_exception(backoff.expo, KlaviyoError, max_tries=3)
+@backoff.on_exception(backoff.expo, (simplejson.scanner.JSONDecodeError, KlaviyoInternalServiceError), max_tries=3)
```

Comment:
@harshpatel4crest Since we have added the backoff for simplejsondecoder error here, can we not remove the same from 'get_incremental_pull' and 'get_full_pulls' functions?

Reply:
done